### DISCUSSION TITLE
Old version worked just fine - the new not

### DISCUSSION THREAD

**Discussion by waswosi** [aclap-dev/video-downloadhelper#144] [Help and Q&A]
### Your question

Doesn't detect some videos which it did before
Download of some videos very slow
when a download was finished there was the video size in the little windows no
there is the file size there again

How can I install the old version?

**Comment by paulrouget** [comment]
> Addon & CoApp version and details
> No response

Please, add details, we can't help you otherwise: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

> Doesn't detect some videos which it did before

Need URL.

**Reply by waswosi** [reply]
Site is watchmdh.to. This site has many ads "covering" it but with the old version it worked very well. I could open many tabs with right click open new tab and somwhere in the file list there was the correct file "Some number ie. 50456" or something.
This doesn't work anymore. Only with one or two tabs.

**Reply by paulrouget** [reply]
Please add details as explained above.

I think your addon is not up to date.

**Reply by waswosi** [reply]
Oh it's Version 8.2.1.1 (stable) and VdhCoApp 2.0.19

**Reply by paulrouget** [reply]
@waswosi Please add details as explained above (click on that [link](https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F) and do exactly what it says).

**Comment by waswosi** [comment]
version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: de
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\SR",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by paulrouget** [comment]
> and somwhere in the file list there was the correct file

There are 2 questions here. Please answer the 2 questions:

1. How do you get that "file list": do you click on the addon icon in the toolbar, or by right clicking on a tab?
2. in the `Settings > More > Video Quality`, if you uncheck `Ignore low quality videos`, do you see the file you're expecting?
